raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate_regression.py

```diff
@@ -528,7 +527,7 @@ def test_regression_onchain_secret_reveal_must_update_channel_state():
     balance_proof = balanceproof_from_envelope(expired_message)
 
     message_identifier = message_identifier_from_prng(pseudo_random_generator)
-    expired_block_number = lock.expiration + DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS * 2
+    expired_block_number = channel.get_receiver_expiration_threshold(lock)
```

Comment:
the expression here was for the sender

Reply:
I disagree. The below state change is handling receiving lock expired so it should validate based on the fact that the current block number is the receiver's